panel/lxqtpanel.h

```diff
@@ -174,6 +174,7 @@ private slots:
 
     ILXQtPanel::Position mPosition;
     int mScreenNum;
+    int mActualScreenNum;
```

Comment:
"Sir" makes me feel like I hope I'll be in my late 40s: a serious, piercing look and a mature, respectable mustache. Unfortunately, it'll take a lot of time.